### PR TITLE
Add .fossa.yml to central-cp-multi-upfs branch for FOSSA CI

### PR DESCRIPTION
**Pull request by sindack**

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: .fossa.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2019-present Open Networking Foundation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (pingping-lin) [inline .fossa.yml]: from now on, we can use the shorter header, can remove this part.

Comment (sindack) [reply .fossa.yml]: Done

**Comment by pingping-lin** [comment]
+1

**Pull request merged**